### PR TITLE
Marketplace remove hash from manifest

### PR DESCRIPTION
**Pull request by NicolasMahe**
Start to implements https://github.com/mesg-foundation/core/issues/861 for CLI and Service Marketplace:
only remove unnecessary hash for now

### CODE REVIEW

**Comment by NicolasMahe** [comment]
@mesg-foundation/core I remove the dependency to the other PR. Ready to review.

**Comment by ilgooz** [comment]
@NicolasMahe I added a fix commit, it's better to squash this one while merging

**Comment by NicolasMahe** [comment]
> @NicolasMahe I added a fix commit, it's better to squash this one while merging

Done

File: systemservices/marketplace/mesg.yml [hunk 1]
```diff
@@ -68,11 +68,7 @@ tasks:
                         type: String	
                       service: &manifestServiceData	
                         type: Object	
-                        object:	
-                          hash:	
-                            name: ""	
-                            description: ""	
-                            type: String	
+                        object:
```
Comment (krhubert) [inline systemservices/marketplace/mesg.yml]:
```suggestion
```

Comment (NicolasMahe) [reply systemservices/marketplace/mesg.yml]: Why do you want to remove this?

Comment (krhubert) [reply systemservices/marketplace/mesg.yml]: Sorry my mistake, on review it looked like empty object

**Pull request merged**